<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QFrame" name="mainframe">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="2" column="0">
        <widget class="QFrame" name="bmp280_frame">
         <property name="minimumSize">
          <size>
           <width>606</width>
           <height>261</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QLabel" name="bmp280_textplace">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
            </property>
            <property name="text">
             <string>temp:
press:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="bmp280_h">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>64</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
            </property>
            <property name="text">
             <string>BMP280</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="PlotWidget" name="bmp280_plotplace" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="ds18b20_frame">
         <property name="minimumSize">
          <size>
           <width>607</width>
           <height>261</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <widget class="QLabel" name="ds18b20_textplace">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
            </property>
            <property name="text">
             <string>temp:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="ds18b20_h">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>64</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
            </property>
            <property name="text">
             <string>DS18B20</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="PlotWidget" name="ds18b20_plotplace" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextBrowser" name="log_frame">
         <property name="minimumSize">
          <size>
           <width>606</width>
           <height>261</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QFrame" name="l3g4200d_frame">
         <property name="minimumSize">
          <size>
           <width>607</width>
           <height>261</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="1" column="1">
           <widget class="QLabel" name="l3g4200d_textplace">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
            </property>
            <property name="text">
             <string>X:
Y:
Z:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="l3g4200d_h">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>64</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
            </property>
            <property name="text">
             <string>L3G4200D</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="PlotWidget" name="l3g4200d_plotplace" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFrame" name="hmc5883ma_frame">
         <property name="minimumSize">
          <size>
           <width>606</width>
           <height>261</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="1">
           <widget class="QLabel" name="hmc5883ma_textplace">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
            </property>
            <property name="text">
             <string>X:
Y:
Z:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="hmc5883ma_h">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>64</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
            </property>
            <property name="text">
             <string>HMC5883MA</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="PlotWidget" name="hmc5883ma_plotplace" native="true">
            <layout class="QGridLayout" name="gridLayout_5"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="adxl345_frame">
         <property name="minimumSize">
          <size>
           <width>607</width>
           <height>261</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="2">
           <widget class="QLabel" name="adxl345_textplace">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
            </property>
            <property name="text">
             <string>X:
Y:
Z:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="adxl345_h">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>64</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FFFFFF;
border-style: solid;
border-width: 1px;
border-color: gray; 
background-color: #000000;</string>
            </property>
            <property name="text">
             <string>ADXL345</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="PlotWidget" name="adxl345_plotplace" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionlogs">
   <property name="text">
    <string>logs</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
